core/src/test/java/com/github/mizool/core/MockitoExtensions.java

```diff
@@ -18,6 +18,9 @@
 
```

Comment:
update copyright year to `2017-2018`

Reply:
right, I forgot about the `NOTICE` file. however, my comment referred to the `MockitoExtensions` boilerplate comment.

Reply:
My bad. Didn't see this because IntelliJ wraps the boilerplate code neatly to a single (unobtrusive) line.